Podfile

```diff
@@ -24,6 +24,7 @@ def defaultPods
     pod 'MaterialComponents/TextFields', '~> 84.0.0'
     pod 'BRCybertron', '~> 1.1.1'
     pod 'MCBinaryHeap', '~> 0.1'
+    pod "TTRangeSlider"
```

Comment:
Check rtl:https://github.com/TomThorpe/TTRangeSlider/issues/95
Add version as in every other pod.